platform/core/src/studies/services/wado/studyInstanceHelpers.js

```diff
@@ -157,6 +158,41 @@ function getFrameIncrementPointer(element) {
   return frameIncrementPointerNames[value];
 }
 
+function getReferencedSeriesSequence(instance) {
```

Comment:
JS Doc?
If private, can we prefix with an underscore?
May be worth kicking to an external file.

Reply:
Done! thanks.